### PR TITLE
documenting MATLAB startup options

### PR DESCRIPTION
**Pull request by mw-hrastega**

### CODE REVIEW

File: CONFIGDOC.md [hunk 1]
```diff
@@ -31,17 +31,17 @@ Once you install the plugin, **Use MATLAB version** appears in the **Build Envir
 
 In this example, the list includes two registered tools as well as the option for specifying a custom installation. If you select **Custom**, a **MATLAB root** box appears in the UI. You must enter the full path to your preferred MATLAB root folder in this box.
 
-![use_matlab_version_tool](https://user-images.githubusercontent.com/48831250/217575011-ab82e8bf-c673-4364-946a-cdd54eb75c39.png)
+![use_matlab_version_tool](https://github.com/mw-hrastega/Times-Table-App/assets/48831250/908748e6-afeb-40c1-a2f8-f2f6bde6f224)
```
Comment (mcafaro) [inline CONFIGDOC.md:34]: Images are linking to your personal repo. Are you planning to change this before merging?

File: CONFIGDOC.md [hunk 2]
```diff
@@ -301,19 +303,20 @@ You can customize the `runMATLABTests` step using optional inputs. For example,
...
 `strict`                  | (Optional) Whether to apply strict checks when running tests, specified as `false` or `true`. By default, the value is `false`. If you specify a value of `true`, the plugin generates a qualification failure whenever a test issues a warning.
 `useParallel`             | (Optional) Whether to run tests in parallel, specified as `false` or `true`. By default, the value is `false` and tests run in serial. If the test runner configuration is suited for parallelization, you can specify a value of `true` to run tests in parallel. This input requires a Parallel Computing Toolbox license.
 `outputDetail`            | (Optional) Amount of output detail displayed for the test run, specified as `'None'`, `'Terse'`, `'Concise'`, `'Detailed'`, or `'Verbose'`. By default, the plugin displays failing and logged events at the `Detailed` level and test run progress at the `Concise` level.
 `loggingLevel`            | (Optional) Maximum verbosity level for logged diagnostics included for the test run, specified as `'None'`, `'Terse'`, `'Concise'`, `'Detailed'`, or `'Verbose'`. By default, the plugin includes diagnostics logged at the `Terse` level.
```
Comment (mcafaro) [inline CONFIGDOC.md:312]: No examples for these?

File: CONFIGDOC.md (hunk 1 above)
Comment (mw-hrastega) [reply CONFIGDOC.md]: Good catch! Fixed.

File: CONFIGDOC.md (hunk 2 above)
Comment (mw-hrastega) [reply CONFIGDOC.md]: Added some inline examples.

**Review by mcafaro — APPROVED** [review]

File: CONFIGDOC.md [hunk 3]
```diff
@@ -362,32 +365,30 @@ pipeline {
...
 
 ```groovy
 // Scripted Pipeline
 node {
-    runMATLABCommand 'myscript'  
+    runMATLABCommand(command: 'myscript')
 }
-``` 
-
-MATLAB exits with exit code 0 if the specified script, function, or statement executes successfully without error. Otherwise, MATLAB terminates with a nonzero exit code, which causes the stage to fail. If you properly react to the resulting MATLAB execution exception, the remaining stages of your pipeline can still run, and your build can succeed. Otherwise, Jenkins terminates the build in the current stage and marks it as a failure.
```
Comment (sameagen-MW) [inline CONFIGDOC.md:381]: I like the change of moving this line into the arguments table!

**Review by sameagen-MW — APPROVED** [review]
Great job, that seems like it was a ton of screenshots to update!

**Pull request merged**